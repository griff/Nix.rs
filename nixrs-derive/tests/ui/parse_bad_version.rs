use nixrs_derive::NixDeserialize;

#[derive(NixDeserialize)]
pub struct Test {
    #[nix(version = 12)]
    version: u8,
}

fn main() {}
