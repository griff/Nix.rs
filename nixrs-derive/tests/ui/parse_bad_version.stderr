error: expected nix attribute version to be string
 --> tests/ui/parse_bad_version.rs
  |
  |     #[nix(version = 12)]
  |                     ^^
